src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.Xaml.cs

```diff
@@ -97,10 +105,8 @@ private bool IsIssue93860Fixed()
 					if (parts.Length > 0 && Version.TryParse(parts[0], out var version))
 					{
 						_isIssue93860Fixed = version >=
-#if __IOS__ || __CATALYST__
-							new Version(17, 2, 8022); // 8.0.200
-#elif __ANDROID__
-							new Version(34, 0, 1, 52); //8.0.102
+#if __ANDROID__
+							new Version(34, 0, 1, 52); // 8.0.102
```

Comment:
code does not look valid if `__ANDROID__` is not defined

Reply:
That's ok for now, this is the only code path that is available. I'm keeping it here to ensure that when we bring iOS back, we'll notice the difference.